src/renderer/windows/main/components/Preferences.vue

```diff
@@ -8,11 +8,15 @@
             <div class="mb-8 mt-8 overflow-y-scroll flex-1 custom-width-for-shadows px-4 -ml-4">
                 <div class="mb-8 flex-1">
                     <label
+                        tabindex="0"
+                        @keyup.enter="changeSection('Manual')"
```

Comment:
MDN suggests to use a keydown event handler: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Keyboard-navigable_JavaScript_widgets#Use_onkeydown_to_trap_key_events_not_onkeypress

Reply:
MDN says: "The expected keyboard shortcut for activating a checkbox is the Space key."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_checkbox_role